### DISCUSSION TITLE
timeseries needs a way to attach right Y-axis related to left Y-axis

### DISCUSSION THREAD

**Discussion by jelmd** [grafana/grafana#42338] [Dashboards, Panels & Visualizations]
For many charts one needs to visualize resources usage as shown in [Learn Grafana: How to use dual axis graphs](https://grafana.com/blog/2020/03/10/learn-grafana-how-to-use-dual-axis-graphs/#resource-utilization). However, the first problem is, that this isn't possible with the new `timeseries` panel. The second problem with this is, that actually there is no need at all to draw a 2nd line (e.g. for %), because it has exactly the same shape/points as the other one (e.g. absolute current value), because this one is the same but multiplied with a fixed factor.

So an efficient panel, just needs to draw e.g. the left-Y axis e.g. with the units for the current values, a single current value line, query the max. value for the shown ts and allow to attach a right Y-axis, which mirrors the left Y-axis, but replaces the shown tics an expression like `$ticValue * 100 / $maxValue`.

So an example wrt. node_exporter:
```
Query A: node_memory_MemAvailable{﻿gz﻿=~﻿"$gz"﻿}
Instant Query B: node_memory_MemTotal﻿{﻿gz﻿=~﻿"$gz"﻿}
    Transform: Hide result
Standard (left-Y-axis) Unit: bytes(IEC)
Right Y-axis: __leftY__ * 100 / $B
    Unit: Percent
```

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
